core/chains/evm/logpoller/log_poller.go

```diff
@@ -1159,3 +1222,13 @@ func EvmWord(i uint64) common.Hash {
 	binary.BigEndian.PutUint64(b, i)
 	return common.BytesToHash(b)
 }
+
+func (lp *logPoller) confirmations(opts BlockConfsOpt) int {
+	if opts.onlyFinalized {
```

Comment:
Actually, it could be `opt.useFinalityDepth` and don't require `useFinalityTag` to be enabled